tests/unit/utils/qb/validation/test_cube.py

```diff
@@ -43,9 +44,9 @@ def test_new_qb_attribute_literal_string_with_template():
     )
 
 
-def test_existing_qb_attribute_resource_no_values():
+def test_new_qb_attribute_resource_no_values():
```

Comment:
Return this test, we should still support existing qb attribute resource with no values (e.g. a csv-w with observations not missing but configured to support observations missing with an obsStatus value)

Reply:
I've left this one as-is, as it's properly testing that the NoUriTemplateOrAttrValuesError is being raised. I've added a new test (test_existing_qb_attribute_resource_no_cell_uri_template), and a new elif to the _validate_attributes function in utils/qb/validation/cube.py to check that an ExistingQbAttribute with no cell_uri_template raises CsvColumnUriTemplateMissingError